package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gpii-no-os",
+  "description": "Stub definitions suitable for booting up the GPII Morphic UI implemented in gpii-app without platform-specific support",
+  "version": "0.3.0",
+  "author": "Raising the Floor -- US Inc.",
+  "bugs": {
+    "url": "http://issues.gpii.net/browse/GPII"
+  },
+  "homepage": "http://gpii.net/",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "gpii-universal": "JavierJF/universal#GPII-3810"
```

Comment:
Why this branch rather than master or, even better, an npm artefact?

Reply:
For consistency, this is the same branch used by the Windows repository.
https://github.com/GPII/windows/blob/master/package.json

I can change it to any other repo/package/branch, as desired.  To date, I have been testing specifically with the branch used by GPII/windows.